org.adempiere.base/src/org/compiere/model/MSystem.java

```diff
@@ -521,4 +521,22 @@ public static boolean isSecureProps() {
 		return true;
 	}
 
+	/**
+	 * The system allows to use login prefix for tenant
+	 * @return
+	 */
+	public static boolean allowLoginPrefix() {
+		String loginWithTenantPrefix = MSysConfig.getValue(MSysConfig.LOGIN_WITH_TENANT_PREFIX, "N");
+		return "F".equals(loginWithTenantPrefix) || "A".equals(loginWithTenantPrefix);
+	}
+
+	/**
+	 * The system forces to use login prefix for tenant
+	 * @return
+	 */
+	public static boolean forceLoginPrefix() {
+		String loginWithTenantPrefix = MSysConfig.getValue(MSysConfig.LOGIN_WITH_TENANT_PREFIX, "N");
+		return "F".equals(loginWithTenantPrefix);
+	}

```

Comment:
While both method looks perfectly fine but I think if by common Java convention, this is probably more natural to be named as IsUseLoginPrefix and isLoginPrefixMandatory.

Reply:
Thanks, done